### DISCUSSION TITLE
RemovedInNextVersionWarning: legacy encryption_mode is deprecated, use modern instead.

### DISCUSSION THREAD

**Discussion by max-bertinetti** [emmett-framework/emmett#417] [Q&A] [answered]
```python
#: pipeline
app.pipeline = [
    SessionManager.cookies('myverysecretkey'),
    db.pipe,
    auth.pipe
]
```

Hi @gi0baro. This is the offending code. What should I do to suppress the warning?

**Comment by gi0baro** [comment, marked as answer]
@eclecticlly you need to install Emmett with `crypto` extras and set the `encryption_mode` parameter of `SessionManager.cookies` to *modern*.

Details available in docs, see:
- https://emmett.sh/docs/2.4.x/sessions#storing-sessions-in-cookies
- https://emmett.sh/docs/2.4.x/upgrading#version-23
